### PR TITLE
Change UI image location to edge-infrastructure org

### PR DESCRIPTION
**Pull request by osherdp**
/cc @nshidlin

### CODE REVIEW

**Comment by osherdp** [comment]
@jkilzi

File: assisted-installer.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ openshift/assisted-service:
 openshift-assisted/assisted-ui:
   revision: b5281fe43c01b2f8adee5483b3bb438bf487cb91
```
Comment (nshidlin) [inline assisted-installer.yaml:11]: does the revision need to be bumped? I ask as you changed this in your last PR

**Comment by osherdp** [comment]
/hold
I'll check if it needs to bump the revision
although we can also just run the jenkins job

**Comment by osherdp** [comment]
@nshidlin updated the revisions as well

**Pull request merged**